delfin/drivers/hds/vsp/alert_handler.py

```diff
@@ -0,0 +1,98 @@
+# Copyright 2020 The SODA Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http:#www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from oslo_log import log
+
+from delfin import exception
+# from delfin.i18n import _
```

Comment:
plz remove commented code. 
In ovrall PR, some code need formatting , like mutiple spaces, extra lines etc. Plz consider it.